test_suite/tests/ui/fail_missing_derive.stderr

```diff
@@ -5,15 +5,15 @@ error[E0277]: the trait bound `PawType<u16>: TypeInfo` is not satisfied
    |                        ^^^^^^^^^^^^^^^^^^ the trait `TypeInfo` is not implemented for `PawType<u16>`
    |
    = help: the following other types implement trait `TypeInfo`:
-             &T
-             &mut T
-             ()
-             (A, B)
-             (A, B, C)
-             (A, B, C, D)
-             (A, B, C, D, E)
-             (A, B, C, D, E, F)
-           and $N others
+             bool
+             char
+             i8
+             i16
+             i32
+             i64
+             i128
+             u8
+           and 57 others
```

Comment:
does this still work?

```suggestion
           and $N others
```

Reply:
Yes but I needed to update the version of `trybuild` first